spec/mailers/user_mailer_spec.rb

```diff
@@ -123,12 +123,6 @@
       expect(mail.html_part.body).to have_content(
         t('user_mailer.phone_changed.intro', app: APP_NAME)
       )
-
-      expect(mail.html_part.body).to have_content(
-        t('user_mailer.phone_changed.help',
-          app: APP_NAME,
-          contact_phone_number: CONTACT_PHONE_NUMBER)
-      )
```

Comment:
If the helper text were mal-formed, this spec would fail whether we are checking for the text or not. (I know because at one point it was!) Given this, perhaps it's OK to not check for the exact help string?